Brokerages/Tradier/TradierBrokerage.cs

```diff
@@ -1001,7 +1000,8 @@ private TradierOrderResponse TradierPlaceOrder(TradierPlaceOrderRequest order)
                     message = "Order " + order.QCOrder.Id + ": " + string.Join(Environment.NewLine, response.Errors.Errors);
                     if (string.IsNullOrEmpty(order.QCOrder.Tag))
                     {
-                        order.QCOrder.Tag = message;
+                        var request = new UpdateOrderRequest(DateTime.UtcNow, order.QCOrder.Id, new UpdateOrderFields { Tag = message });
+                        order.QCOrder.ApplyUpdateOrderRequest(request);
```

Comment:
hm not sure this is a good idea, believe `UpdateOrderRequest` were meant to be are user driven order updates. How is this done by other brokerages?